src/main/java/core/basesyntax/service/CsvReader.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class CsvReader implements Readable {

```

Comment:
this class can read not only csv files
```suggestion
public class FileReader implements Readable {
```